### ISSUE TITLE
Dialog routeName not announced by TalkBack

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#19901]

**Comment by jonahwilliams** [comment]
I can't reproduce this anymore, not sure if it was an issue caused by a local build..
